docs/install.rst

```diff
@@ -10,10 +10,7 @@ Requirements
 
 * Supported OS: Tutor runs on any 64-bit, UNIX-based OS. It was also reported to work on Windows (with `WSL 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`__).
 * Architecture: support for ARM64 is a work-in-progress. See `this issue <https://github.com/overhangio/tutor/issues/510>`__.
-* Required software:
-
-    - `Docker <https://docs.docker.com/engine/installation/>`__: v20.10.15+
-    - `Docker Compose <https://docs.docker.com/compose/install/>`__: v2.0.0+
+* Required software: `Docker <https://docs.docker.com/engine/installation/>`__ v20.10.15+.
```

Comment:
We still need to specify the minimum Compose version, because Compose does not always ship with Docker v20.

Reply:
Good catch, fixed.